### DISCUSSION TITLE
How to use domain name + port instead of only domain name?

### DISCUSSION THREAD

**Discussion by Zieng** [nextcloud/all-in-one#5604] [Questions] [answered]
The problem is that my ISP blocks 80 and 443 port, so using `https://some_domain.com` can not access my machine. Instead I need to use something like `https://some_domain.com:10443` and forward the 10443 to port 443 of my machine on the router.

So I'm wondering does nextcloud aio supports to use the `domain_name:ip` instead of `domain_name`?

**Comment by szaimen** [comment, marked as answer]
Hi, see https://github.com/nextcloud/all-in-one?tab=readme-ov-file#are-other-ports-than-the-default-443-for-nextcloud-supported

**Reply by Zieng** [reply]
Thanks! I saw that later after I've posted the question. Sorry to forget to close it. Nextcloud AIO is great but due to my network limitation I've to try something else. Hope the project will be better and better!

**Reply by szaimen** [reply]
As mentioned in the docs, you could use a Cloudflare Tunnel: https://github.com/nextcloud/all-in-one?tab=readme-ov-file#how-to-run-nextcloud-behind-a-cloudflare-tunnel
